components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -2196,7 +2193,7 @@ private OMElement queryBetweenTwoDaysForAPIUsageByVersion(String columnFamily, S
         }
     }
 
-    private OMElement queryBetweenTwoDaysForAPIUsageByUser(String providerName, String fromDate, String toDate, Integer limit)
+    private List<APIUsageByUserName> queryBetweenTwoDaysForAPIUsageByUser(String providerName, String fromDate, String toDate, Integer limit)
```

Comment:
rename method as "getAPIUsageByUserData" and fixed in commit: https://github.com/wso2/carbon-apimgt/commit/006d2e055000123aa5f1e15f2b21a85b8003b621